pkg/webhooks/sgx/sgx.go

```diff
@@ -190,7 +208,7 @@ func (s *SgxMutator) Handle(ctx context.Context, req admission.Request) admissio
 		return admission.Errored(http.StatusInternalServerError, err)
 	}
 
-	return admission.PatchResponseFromRaw(req.Object.Raw, marshaledPod)
+	return admission.PatchResponseFromRaw(req.Object.Raw, marshaledPod).WithWarnings(warnings...)
```

Comment:
What happens here if `warnings` is nil? I'm just wondering if we still get empty warnings printed out or similar.

Reply:
Nothing happens:
```
$ kubectl run nginx --image=nginx --limits 'sgx.intel.com/enclave=1'
Warning: sgx.intel.com/enclave should not be used in Pod spec directly
pod/nginx created
$ kubectl delete pod nginx
pod "nginx" deleted
$ kubectl run nginx --image=nginx
pod/nginx created
```